src/CMakeLists.txt

```diff
@@ -3,10 +3,19 @@ list(REMOVE_ITEM src_files ${CMAKE_CURRENT_SOURCE_DIR}/main.cpp)
 
 find_package(GSL REQUIRED)
 
+# Fetch the TCLAP library from the git repository and place it
+# into the source folder under src/tclap
+IF(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/tclap )
+    execute_process( COMMAND git archive --remote=git://git.code.sf.net/p/tclap/code HEAD:include -o tclap.zip )
+    execute_process( COMMAND unzip -d ${CMAKE_CURRENT_SOURCE_DIR} tclap.zip )
+    execute_process( COMMAND rm -rf tclap.zip ${CMAKE_CURRENT_SOURCE_DIR}/Makefile.am )
+ENDIF()
+
 set (PROJECT_LIBS ${PROJECT_NAME}-lib)
 add_library(${PROJECT_LIBS} SHARED ${src_files})
 target_link_libraries(${PROJECT_LIBS} PUBLIC GSL::gsl GSL::gslcblas)
-target_include_directories(${PROJECT_LIBS} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(${PROJECT_LIBS} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I don;t think that this change is necessary. The `src` directory is already part of the header search path, so with the relative path `<tclap/...` in the source files, everything should be found with the original settings.